lib/Model/Importer/DebugInfo/PDBImporter.cpp

```diff
@@ -764,18 +731,16 @@ Error PDBImporterTypeVisitor::visitKnownRecord(CVType &Record,
 
         auto &FieldType = Struct->Fields()[Offset];
         FieldType.OriginalName() = Field.getName().str();
-        QualifiedType TheUnderlyingType(*FiledTypeFromModel, {});
-        FieldType.Type() = TheUnderlyingType;
+        FieldType.Type() = std::move(FieldModelType);
```

Comment:
You're introducing quite a few `std::move`. Please run the tests with ASan `:P`

Reply:
Done!